predict.py

```diff
@@ -76,6 +88,10 @@ def predict(
             description="Specify things to not see in the output",
             default=None,
         ),
+        image: Path = Input(
+            description="Inital image to generate variations of. This enables img2img! Ensure your images are reasonably sized as the output is the same size as your input image (height and width input parameters are ignored).",
```

Comment:
Does "reasonably sized" mean the images should have dimensions that match the available input widths and heights?

Reply:
hmm, no

"as the output is the same size as your input image" is trying to say if you send an image that is 1024x1024, the output will be the same exact size.

It is why @chenxwh removed `height` / `width` from img2img for stable diffusion:

https://github.com/replicate/cog-stable-diffusion-img2img/pull/2

Reply:
I don't have better recommendations for "reasonable size" as it depends on the GPU you are running on :-/

Reply:
Gotcha. I've suggested a change to the description.